python/cudf/cudf/core/column/column.py

```diff
@@ -15,32 +15,186 @@
 import cudf
 import cudf._lib as libcudf
 from cudf._lib.stream_compaction import nunique as cpp_unique_count
+from cudf._libxx.column import Column
 from cudf.core.buffer import Buffer
+from cudf.core.dtypes import CategoricalDtype
 from cudf.utils import cudautils, ioutils, utils
-from cudf.utils.dtypes import is_categorical_dtype, is_scalar, np_to_pa_dtype
-from cudf.utils.utils import buffers_from_pyarrow
+from cudf.utils.dtypes import (
+    is_categorical_dtype,
+    is_scalar,
+    is_string_dtype,
+    np_to_pa_dtype,
+)
+from cudf.utils.utils import (
+    buffers_from_pyarrow,
+    calc_chunk_size,
+    mask_bitsize,
+)
+
+
+class ColumnBase(Column):
+    def __init__(
+        self, data, size, dtype, mask=None, offset=0, name=None, children=()
+    ):
+        """
+        Parameters
```

Comment:
Could use a brief description of this class above this.

Reply:
Done